### ISSUE TITLE
Call to undefined method App\RedisModels\User::preventsAccessingMissingAttributes()

### ISSUE THREAD

**Issue by bnnoor** [alvin0/redis-model#16]
when i want get anything: 

Call to undefined method App\RedisModels\User::preventsAccessingMissingAttributes()

User::all()
or 
User::get();
or 
User::where.....

**Comment by bnnoor** [comment]
User Model : 
```
class User extends Model
{
    /**
     * The primary key for the model.
     *
     * @var string
     */
    protected $primaryKey = 'id';

    /**
     * The model's sub keys for the model.
     *
     * @var array
     */
    protected $subKeys = ['username'];

    /**
     * The attributes that should be cast.
     *
     * @var array
     */
    protected $casts = [];

    /**
     * The attributes that are mass assignable.
     *
     * @var array<string>
     */
    protected $fillable = [
        'id',
        'name',
        'last_name',
        'username'
    ];
}
```

**Comment by alvin0** [comment]
Hi @bnnoor ,

Please provide me with more information to reproduce the error you encountered. Currently, I am using it on Laravel version 11, and I do not see the issue.

Additionally, I have released version 1.0 of the package to ensure compatibility with Laravel versions 10 and 11. It should work well with PHP version 8.1 and above.

![image](https://github.com/user-attachments/assets/a6388507-a565-4789-b519-d3bc7339a500)
![image](https://github.com/user-attachments/assets/6e3ba18c-68a6-45e3-bb9f-7e2b7434c940)
![image](https://github.com/user-attachments/assets/6c54ad67-76eb-480c-a792-8ba669293c07)
